src/frontend/extract-helpers/get-reconciliation-rows.ts

```diff
@@ -44,10 +44,7 @@ export const getReconciliationRows = (
         item.billing_price_formatted,
         item.price_difference_percentage
       ),
-      billingPriceInclVat: getPrice(
-        item.billing_amount_with_tax,
-        item.price_difference_percentage
-      ),
+      billingPriceInclVat: item.billing_amount_with_tax,
```

Comment:
When invoice data is missing, now in Invoice table Total+Vat column  is empty instead of "Invoice data missing". I think we need to be consistent in showing the message in Invoice table similar to other cell values in that table. 
![image](https://github.com/alphagov/di-billing-transaction-monitoring/assets/112889496/4b3a14d7-fd3f-4cbc-ac13-ce5250a192d4)